src/Integration.Vsix.UnitTests/Analysis/SingleFileMonitorTests.cs

```diff
@@ -381,85 +382,72 @@ public void Real_File_ChangesToOtherFiles_AreIgnored()
             }
         }
 
-        private static Mock<IFileSystemWatcherFactory> CreateFactoryAndWatcherMocks(out Mock<IFileSystemWatcher> watcherMock)
-        {
-            watcherMock = new Mock<IFileSystemWatcher>();
-            var watcherFactoryMock = new Mock<IFileSystemWatcherFactory>();
-            watcherFactoryMock
-                .Setup(x => x.CreateNew())
-                .Returns(watcherMock.Object);
-            return watcherFactoryMock;
-        }
 
-        #endregion Simple file operations
-
-        [Ignore] // Flaky -> disabled https://github.com/SonarSource/sonarlint-visualstudio/issues/3236
         [TestMethod]
-        public void RealFile_MultipleOperations_NoDuplicates()
+        public void File_DuplicateChangesSameTime_AreIgnored()
```

Comment:
This and the other test I added are basically identical except for the dateTime.AddMillisecond(1)